sphinx/source/_static/custom.css

```diff
@@ -0,0 +1,227 @@
+
+code {
+    font-family: SFMono-Regular,Menlo,Monaco,Consolas,Liberation Mono,Courier New,monospace;
+    color: rgb(33, 37, 41);
+    padding: 2px 1px 0px 1px;
+    font-size: 80%;
+    background-color:#eef0f1;
+}
+
+/* Customize theme font colors */
+:root {
+    --pst-color-primary: 33, 37, 41;
+    --pst-color-success: 165, 205, 57;
+    --pst-color-info: 0, 170, 174;
+    --pst-color-warning: 240, 82, 35;
+    --pst-color-danger: 220, 53, 69;
+    --pst-color-text-base: var(--pst-color-primary);
+
+    --pst-color-h1: var(--pst-color-primary);
+    --pst-color-h2: var(--pst-color-primary);
+    --pst-color-h3: var(--pst-color-text-base);
+    --pst-color-h4: var(--pst-color-text-base);
+    --pst-color-h5: var(--pst-color-text-base);
+    --pst-color-h6: var(--pst-color-text-base);
+    --pst-color-paragraph: var(--pst-color-text-base);
+    --pst-color-link: 255, 0, 0;
+    --pst-color-link-hover: 227, 46, 0;
+    --pst-color-headerlink: 198, 15, 15;
+    --pst-color-headerlink-hover: 255, 255, 255;
+
+    --pst-color-active-navigation: var(--pst-color-primary);
+    --pst-color-navbar-link: 77, 77, 77;
+    --pst-color-navbar-link-hover: var(--pst-color-active-navigation);
+    --pst-color-navbar-link-active: var(--pst-color-active-navigation);
+    --pst-color-sidebar-link: 77, 77, 77;
+    --pst-color-sidebar-link-hover: var(--pst-color-active-navigation);
+    --pst-color-sidebar-link-active: var(--pst-color-active-navigation);
+    --pst-color-sidebar-caption: 77, 77, 77;
+    --pst-color-toc-link: 119, 117, 122;
+    --pst-color-toc-link-hover: var(--pst-color-active-navigation);
+    --pst-color-toc-link-active: var(--pst-color-active-navigation);
+
+    --pst-color-admonition-default: var(--pst-color-info);
+    --pst-color-admonition-note: var(--pst-color-info);
+    --pst-color-admonition-attention: var(--pst-color-warning);
+    --pst-color-admonition-caution: var(--pst-color-warning);
+    --pst-color-admonition-warning: var(--pst-color-warning);
+    --pst-color-admonition-danger: var(--pst-color-danger);
+    --pst-color-admonition-error: var(--pst-color-danger);
+    --pst-color-admonition-hint: var(--pst-color-success);
+    --pst-color-admonition-tip: var(--pst-color-success);
+    --pst-color-admonition-important: var(--pst-color-success);
+
+    --pst-icon-admonition-default: var(--pst-icon-info-circle);
+    --pst-icon-admonition-note: var(--pst-icon-info-circle);
+    --pst-icon-admonition-attention: var(--pst-icon-exclamation-circle);
+    --pst-icon-admonition-caution: var(--pst-icon-exclamation-triangle);
+    --pst-icon-admonition-warning: var(--pst-icon-exclamation-triangle);
+    --pst-icon-admonition-danger: var(--pst-icon-exclamation-triangle);
+    --pst-icon-admonition-error: var(--pst-icon-times-circle);
+    --pst-icon-admonition-hint: var(--pst-icon-lightbulb);
+    --pst-icon-admonition-tip: var(--pst-icon-lightbulb);
+    --pst-icon-admonition-important: var(--pst-icon-exclamation-circle);
```

Comment:
(sidenote: I would personally recommend to not include all those variables that were not adapted compared to the base theme, then it will be easier to keep track of what is actually customized here)

Reply:
cc @tcmetzger on this I don't have a preference